docs/install-pro.md

```diff
@@ -0,0 +1,56 @@
+# Install Pro packages of Percona Server for MySQL
+
+This document provides guidelines how to install Pro packages of Percona Server for MySQL from Percona repositories. [Learn more about Percona Server for MySQL Pro :material-arrow-top-right:](../psmysql-pro.md).
+
+## Procedure
+
+1. Request the access to the pro repository from Percona Support. You will receive the client ID and the access token.
```

Comment:
client ID and access token which you use when downloading the packages.

Reply:
Updated